src/controllers/objecting.entity.name.controller.ts

```diff
@@ -16,6 +22,67 @@ const validators = [
   check(DIVULGE_INFO_FIELD).not().isEmpty().withMessage(ErrorMessages.SELECT_TO_DIVULGE),
 ];
 
+const getSessionPageDataIfPresent = (session: Session, res: Response) => {
```

Comment:
wondering if we could improve the 2 new function names - the getSessionPageData... makes me think the function is going to return some data from the session we can use, but in reality it is showing the page with the session data. So, how about naming it showPageWithSessionDataIfPresent? - similar for getExistingPageData -> showPageWithMongoData perhaps?

Reply:
Names changed.